Dockerfile.ocp

```diff
@@ -4,8 +4,15 @@ COPY . .
 RUN make build
 RUN make tools
 
+COPY /go/src/github.com/metal3-io/baremetal-operator/config/crd/ocp_kustomization.yaml /go/src/github.com/metal3-io/baremetal-operator/config/crd/kustomization.yaml
```

Comment:
I was thinking if it could be better to have a dedicated, not ambiguous folder for storing OpenShift specific customizations (in this case, just the `ocp_kustomization.yaml` file) in the project, ie something like `/ocp`, in order to reduce the chance of accidental deletion/overvwrite during upstream imports